documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -14,13 +14,58 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 .Procedure
 
-. Upgrade the {operator-name}.
+. Change the update channel:
 +
 ifeval::["{build}" == "upstream"]
-See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
 endif::[]
 ifeval::["{build}" == "downstream"]
-See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-upgrading-operators[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
+endif::[]
+
+. Restart the `CatalogSource` pod:
+
+.. In the the {ocp-short} web console, navigate to the *Operators* -> *Installed Operators* -> *Forklift Operator* -> *Operator details* page. On the *Subscriptions* tab, note the name in the *CatalogSource* field.
+..  From the command line, run the following command:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-marketplace
+----
+Example output:
++
+[source,terminal]
+----
+NAME                                                              READY   STATUS      RESTARTS   AGE
+56743631b0a246d54b96aaca5d217c1221516ae0359bd69295316c--1-j2vsd   0/1     Completed   0          19h
+7fd3c8520985a7090567f60e97203dfbb6da3a051f1bbc06334e3e--1-n5pvq   0/1     Completed   0          19h
+certified-operators-64bn2                                         1/1     Running     0          34h
+community-operators-97cct                                         1/1     Running     0          15h
+hco-catalogsource-bcf5b                                           1/1     Running     0          4d3h
+marketplace-operator-7744f76566-7h9lg                             1/1     Running     0          8m42s
+ocs-catalogsource-cjdf6                                           1/1     Running     0          4d3h
+redhat-marketplace-8nwq5                                          1/1     Running     0          2d14h
+redhat-operators-mgrrm                                            1/1     Running     0          19h
+----
+
+..  Locate the CatalogSource pod.
+..  Run the following command:
++
+[source,terminal]
+----
+$ oc delete pod -n openshift-marketplace <pod_name>
+----
+On the *Subscriptions* tab of the *Operator details* page in the {ocp-short} web console, *Upgrade status* changes from *Up to date* to *Upgrade available*.
+
+. If you have set *Update approval* on the *Subscriptions* tab to *Automatic*, the upgrade starts automatically.
++
+If you have set *Update approval* on the *Subscriptions* tab to *Manual*, approve the upgrade:
```

Comment:
have set > set
Remove the colon at the end of this sentence. Colons introduce lists or diagrams or tables, not other sentences.